framework/src/modules/chain/blocks/block.js

```diff
@@ -134,10 +134,11 @@ const objectNormalize = (block, exceptions = {}) => {
 			delete block[key];
 		}
 	});
-	try {
-		validator.validate(blockSchema, block);
-	} catch (schemaError) {
-		throw schemaError.errors;
+
+	validator.validate(blockSchema, block);
+
+	if (validator.errors) {
+		throw validator.errors;
```

Comment:
A lot of redundant if conditions introduced. Validator must throw error in case of failure.

Reply:
I think this is outside the scope of this PR for which only methods that were missing were added to this library. Please open an issue with suggestions for lisk-validator